apps/src/templates/EligibilityChecklist.jsx

```diff
@@ -0,0 +1,223 @@
+/** @file Maker Discount Code Eligibility Checklist */
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import Button from "./Button";
+import SchoolAutocompleteDropdown from './SchoolAutocompleteDropdown';
+
+import SetupStep, {Status} from '../lib/kits/maker/ui/SetupStep';
+
+export default class EligibilityChecklist extends Component {
+  static propTypes = {
+    statusPD: PropTypes.oneOf(Object.values(Status)).isRequired,
+    statusStudentCount: PropTypes.oneOf(Object.values(Status)).isRequired,
+  };
+
+  state = {
+    statusYear: Status.UNKNOWN,
+    yearChoice: false, // stores the teaching-year choice until submitted
```

Comment:
`yearChoice` as a bool still feels a little bit strange to me (the variable name makes me expect a year not a bool). At the least, I wonder if we might want to change this to something like `eligibleYearChoice`. We may end up wanting to save their actual choice though (this might be data we want in our db). That all said, I think this is probably fine for now and we can potentially change it in the future.